lessons/translating-words-math/langs/en-us/index.adoc

```diff
@@ -0,0 +1,176 @@
+= Translating Between Words and Math
+
+@description{Students learn to model arithmetic expressions in words with a visual tool known as "Circles of Evaluation".}
+
+@lesson-prereqs{}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Model an arithmetic expression using @vocab{Circles of Evaluation}.
+* Translate between expressions in words, Circles of Evaluation, and mathematical expressions.
+* Recognize when expressions in words are ambiguous.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions in words and numbers.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* This activity involves a card sort. The easiest way to prepare for this is to give each pair of students an envelope containing the three sets of cards. Keep each set (A, B, and C) together with rubber bands or paperclips.
+
+| Key Points For The Facilitator
+|
+* This lesson may challenge students' ideas of math as a subject that is entirely black and white. This theme - that there are oftentimes a variety of completely valid ways of seeing a seemingly simple problem - will emerge again and again throughout the course.
+|===
+
+== Circles of Evaluation @duration{30 minutes}
+
+=== Overview
+Students match Circles of Evaluation to mathematical expressions, and then they consider how mathematical expressions in words map onto Circles of Evaluation.
+
+=== Launch
+
+Explain to students that they are going to receive @handout{pages/three-sets-cards.adoc, "three sets of cards"}.
+
+The first set of cards includes mathematical expressions. Students have definitely encountered those before! The expressions on their cards each include the number 15, the number 3, and an operator (+, -, *, or /).
+
+The second set of cards includes _Circles of Evaluation_. Students have almost definitely never encountered Circles of Evaluation before! Broadly, Circles of Evaluation will help students to visualize the structure of the mathematical expressions they encounter. For the time being, here’s what students need to know about Circles of Evaluation:
+
+- Every Circle must have one - and only one! - function, written at the top.
```

Comment:
As the student, I would need to be told from the beginning that circles will be used to model much more complex expressions with nested circles in order to get me to buy into them.